packages/components/Molecules/Card/CardKeydates.vue

```diff
@@ -30,6 +31,24 @@ export default {
   components: {
     RplCardContent,
     RplIcon
+  },
+  computed: {
+    keydatesTrimed: function () {
+      let trimedKeyDates = this.keydates
+      const titleMaxLength = 80
+      const titleMinLength = 40
+      const descriptionLength = 120
+      if (typeof window !== 'undefined') {
+        if (this.keydates.length > 1) {
+          trimedKeyDates = this.keydates.map(dates => ({
+            date: dates.date,
+            title: (dates.description.length > titleMaxLength && dates.title.length > titleMinLength) ? truncateText(dates.title, titleMinLength) : truncateText(dates.title, titleMaxLength),
+            description: dates.description.length > descriptionLength ? truncateText(dates.description, descriptionLength) : dates.description
```

Comment:
This could just be `truncateText(dates.description, descriptionLength)` as the function will return full string if it doesn't reach the defined length.